jerry-main/main-unix.c

```diff
@@ -50,6 +50,25 @@
 
 static uint8_t buffer[ JERRY_BUFFER_SIZE ];
 
+#ifdef JERRY_PORT_ENABLE_EXTERNAL_CONTEXT
```

Comment:
Can I update it after receiving enough comments on the external context api/implementation?
I guess there should be lots of review comments on that.